.gitignore

```diff
@@ -23,6 +23,8 @@ bld/
 
 # Visual Studio 2015 cache/options directory
 .vs/
+# Visual Studio Code / options directory
+.vscode
```

Comment:
This should not be ignored. This folder is from VS Code to hold workspace specific settings.